lsp-go.el

```diff
@@ -143,17 +169,15 @@ defaults to half of your CPU cores."
                   :priority -1
                   :initialization-options 'lsp-clients-go--make-init-options
                   :server-id 'go-bingo
-                  :library-folders-fn (lambda (_workspace)
-                                        lsp-clients-go-library-directories)))
+                  :library-folders-fn 'lsp-clients-go--library-default-directories))
```

Comment:
Does bingo support this new feature?

Reply:
Bingo does seem to have support for go modules (it itself is implemented as a module).